riff-raff/app/deployment/Deployments.scala

```diff
@@ -157,18 +157,16 @@ class Deployments(deploymentEngine: DeploymentEngine, builds: Builds)(implicit v
 
   def cleanup(uuid: UUID) {
     log.debug(s"Queuing removal of deploy record $uuid from internal caches")
-    library sendOff { allDeploys =>
```

Comment:
Why was this changed to `send` instead of `sendOff`? The later had non-blocking semantics IIRC?

Reply:
`send` and `sendOff` were both non-blocking, but `sendOff` put the work on a different thread (iirc). That mean adding a lot of complexity to support it in Box, so it isn't there at the moment. This was basically seeing if the simplest thing that might possibly work did work.